engine/runtime-with-instruments/src/test/java/org/enso/interpreter/test/instrument/AvoidIdInstrumentationTagTest.java

```diff
@@ -77,6 +79,33 @@ public void avoidIdInstrumentationInLambdaMapFunctionWithNoise() throws Exceptio
     assertAvoidIdInstrumentationTag(isLambda);
   }
 
+  @Test
+  public void avoidIdInstrumentationInLambdaMapFunctionYear2010() throws Exception {
```

Comment:
There should be more tests ideally, otherwise I'm afraid the issue will keep reappearing

Reply:
The question is: how can we get such tests? Shall we type them randomly? Copy existing ones and just change bits of them? Will that lead to a new test that'd fail? Or shall we wait for a new bug report from the real IDE usage? Can we detect _too frequent_ updates and encourage submitting bugreport rather than getting indirect _too many DB updates_ error?